addons/plugin_refresher/refresh_button.gd

```diff
@@ -174,15 +173,15 @@ func _search_dir_for_plugins(relative_base_folder: String = ""):
 	var dir := DirAccess.open(path)
 	
 	for subdir_name in dir.get_directories():
-		var relative_folder = relative_base_folder.path_join(subdir_name)
+		var relative_folder := relative_base_folder.path_join(subdir_name)
 		var subdir := DirAccess.open(path.path_join(subdir_name))
 		if subdir == null: # Can happen for symlink. They are listed as folder, but if the link is broken, DirAccess returns null
 			continue
 		for file in subdir.get_files():
 			if file == "plugin.cfg":
-				if plugin_folder.path_join(relative_folder) == refresh_plugin.get_script().resource_path.get_base_dir():
+				if plugin_folder.path_join(relative_folder) == (refresh_plugin.get_script() as Script).resource_path.get_base_dir():
```

Comment:
The method "get_base_dir()" is not present on the inferred type "Variant" (but may be present on a subtype).